client/app/components/packages/pas-form/edit.hbs

```diff
@@ -741,19 +727,14 @@
                   </button>
                 </div>
                 <div class="cell large-6 text-right">
-                  <button
-                    type="button"
-                    class="button expanded large no-margin"
-                    disabled={{this.submit.isRunning}}
-                    {{on "click" (perform this.submit)}}
+                  <Packages::save-button
```

Comment:
I think the naming should be camelCase `Packages::SaveButton`

Reply:
Oops. Fixed in 22c8602.